src/services/offline/MockModels.ts

```diff
@@ -50,8 +51,8 @@ export const OfflineNetworkProperties = {
         new ChainProperties(
             true,
             true,
-            networkConfig[NetworkType.TEST_NET].networkConfigurationDefaults.currencyMosaicId,
-            networkConfig[NetworkType.TEST_NET].networkConfigurationDefaults.harvestingMosaicId,
+            '0x' + networkConfig[NetworkType.TEST_NET].networkConfigurationDefaults.currencyMosaicId,
```

Comment:
why we need '0x' here?

Reply:
That is an issue for offline tx showing mosaic (xym) Expired.

because the mosaic id was `.substring(2)` in the `configurationStringAsString` method.

So added `0x` in the mock is to match mosaic id from chain network property
![image](https://user-images.githubusercontent.com/5649156/123787548-88ed7e00-d90d-11eb-971f-2ec7fb10a566.png)